.travis.yml

```diff
@@ -50,20 +51,24 @@ matrix:
       env:
         - MATRIX_EVAL="CXX=clang++-3.8"
           CXXFLAGS="-stdlib=libc++"
+          CXX_TYPE="clang"
           PARALLEL=2
     - <<: *linux_gcc
       env:
         - MATRIX_EVAL="CXX=g++-4.9"
+          CXX_TYPE="gcc"
```

Comment:
Or this?

Reply:
same thing here. Let me check.

Reply:
Ah, I now see that these were in .travis.yml which has since been deleted. Sorry for wasting your time here :)

Reply:
NP... it was a fast fix for me.